src/message.ts

```diff
@@ -1,5 +1,5 @@
 import type { Peer } from "./peer.ts";
-import { randomUUID } from "uncrypto";
+import { randomUUID } from "node:crypto";
```

Comment:
We should avoid `node:` dependencies. We can use `globalThis.crypto.randomUUID` instead and fallback to simple math.random + warning if not available)

Reply:
Can you clarify the reason to avoid `node:` dependencies? I believe they're still supported on non-Node platforms like Deno

Reply:
Cloudflare workers in particular do not support `node:` imports (unless Node.js compatibility is turned on) -- and it might be the case for future runtimes.

We have a decent web standard to depend on, only limit is Node.js <= 18 support which uncrypto [7.69kB](https://packagephobia.com/result?p=uncrypto) solved ecosystem-wide since two years ago using export conditions :)

I agree we could now avoid dependency since Node.js 18 is getting EOL very soon however for cross runtime servers I always avoid depending on `node:` specifiers.

There are modern alternatives, export conditions and `process.getBuiltinModule` if in the future we want to optionally use a Node.js native functionality in core.

Reply:
I don't do a ton with Cloudflare, but as I recall, Cloudflare Workers have to be bundled before deployment and the bundling step generally removes the `node:` import

I'd love to require Node 20, but unfortunately that would block us from using new versions of the library until the next major of SvelteKit as we still support Node 18 for the foreseeable future. Maybe we just close this PR for now and revisit it at that time?

Reply:
wrangler can add node compatibility if enabled and in bundle mode. In other cases (when noBundle flag is used for custom framework and vite plugin) it doesn't do anything.

If you also need to keep Node 18 as minimum for now, let's keep it then and revisit it later (18 is out of support in less than two months anyway!)

Reply:
It'll probably be more than two months in our side, but looking forward to it!